src/Raven.Server/Documents/TimeSeries/TimeSeriesStats.cs

```diff
@@ -70,7 +70,10 @@ public TimeSeriesStats(Transaction tx)
         private Table GetOrCreateTable(Transaction tx, CollectionName collection)
         {
             var tableName = collection.GetTableName(CollectionTableType.TimeSeriesStats); // TODO: cache the collection and pass Slice
-            TimeSeriesStatsSchema.Create(tx, tableName, 16);
+            
+            if (tx.IsWriteTransaction)
```

Comment:
I do not like this. Better to have 2 separate methods. Can't we have 2 separate methods? GetTable and GetOrCreateTable?